src/skills-builder/skills-builder-modal/SkillsBuilderModal.jsx

```diff
@@ -28,14 +29,39 @@ const SkillsBuilderModal = () => {
     history.goBack();
   };
 
+  const sendActionButtonEvent = (eventSuffix) => {
+    sendTrackEvent(
+      `edx.skills_builder.${eventSuffix}`,
+      {
+        category: 'skills_builder',
+        learnerData: {
+          ...state,
```

Comment:
Given that we're sending all of the `state` when the user clicks `exit` or `close`, do we know what all would be sent other than `currentGoal`, `currentJob`, and `careerInterests`? Do we want to be more explicit?

Reply:
Those are the only values currently being stored in `state`. We could extract them here to be more explicit if we feel it's necessary.

I would think that anything else we end up storing in `state` would also be relevant to the event, but there's no way to be sure. It's easy enough to extract them as we are already extracting `careerInterests` in this same file. I'll make that change 👍

Reply:
The one thing I worry about with being explicit on the properties we want to send out for `exit` and `close` is that _if_ the user decides to leave the quiz _before_ they finish, will there be any complaints from `sendTrackEvent` for having empty/non-existent properties. If you can account for that, this should work.

Reply:
Good catch! Just double checked to be sure that Segment is okay to receive empty strings as values. Each of the values in state default to an empty string, so they will all be defined when we go to `sendTrackEvent` 👍